### PR TITLE
Commercial section and Sub0 link

### PR DESCRIPTION
**Pull request by ruslantalpa**

### CODE REVIEW

**Comment by ruslantalpa** [comment]
Can't figure out why test fail.
Locally i get the string `Voil?` as problem, but i see it's in the dict as `Voilà`

File: docs/install/ecosystem.md [hunk 1]
```diff
@@ -24,3 +24,7 @@
 ### In Production
 
 * [Catarse](https://www.catarse.me/)
+
+### Commercial 
+
+* [Sub0](http://graphqlapi.com) - Automated GraphQL & REST API for your PostgreSQL database with builtin caching
```
Comment (begriffs) [inline docs/install/ecosystem.md]: The spell checker is complaining about `builtin`. I'd suggest `built-in`.

**Comment by ruslantalpa** [comment]
Thank you, that worked.
I just assumed `builtin` was an output by CircleCI ...

**Comment by ruslantalpa** [comment]
thank you

**Comment by begriffs** [comment]
No problem. I have to run a script to actually deploy to GH pages, so it won't appear immediately on the site. I'd like to set up continuous deployment through Circle CI by doing something like this https://mwop.net/blog/2016-01-29-automating-gh-pages.html

**Comment by ruslantalpa** [comment]
wow, lots of code in that article :)
i've been recently trying cloud.docker.com
it goes like code -> github -> automated container build -> deploy container on nodes
once you have the container, quite easy to setup autodeploy in the cloud of your choosing

**Pull request merged**